src/sb_utils.R

```diff
@@ -29,4 +46,42 @@ sb_render_post_xml <- function(sb_id, ..., xml_file = NULL){
   
   sb_replace_files(sb_id = sb_id, xml_file)
   
-}
\ No newline at end of file
+}
+
+# Helper function to create a task_table for the files that need to be pushed to SB
+do_item_replace_tasks <- function(files, sb_id, task_yml, out_dir) {
+  # Define task table rows
+  tasks <- basename(files)
+  
+  # Define task table columns
+  sb_push <- scipiper::create_task_step(
+    step_name = 'push_file_to_sb',
+    target_name = function(task_name, step_name, ...){
+      sprintf("%s_pushed_to_sb", task_name)
+    },
+    command = function(task_name, ...){
+      sprintf("item_replace_files('%s', '%s')", sb_id, task_name)
```

Comment:
I think this needs to be 
`sprintf("item_replace_files(I('%s'), '%s')", sb_id, task_name)` (note the `I()`)